data/rule_data.yml

```diff
@@ -198,3 +198,16 @@ rule_data:
   - 2025-12-30
   - 2025-12-31
   - 2026-01-01
+
+  # Usage: https://enterprisecontract.dev/docs/ec-policies/release_policy.html#olm__required_olm_features_annotations_provided
+	required_olm_features_annotations:
```

Comment:
This looks off. Is it a tab? That's probably why the tests are failing.

Reply:
Whoops. Was an issue with my vscode editor. It should be fixed now.